ractor.rb

```diff
@@ -184,16 +202,6 @@
 #     # GOOD=good
 #     # can not access non-shareable objects in constant Object::BAD by non-main Ractor. (NameError)
 #
-#     # Consider the same C class from above
-#
-#     r = Ractor.new do
-#       puts "I see #{C}"
-#       puts "I can't see #{C.tricky}"
-#     end
-#     r.take
-#     # I see C
-#     # can not access instance variables of classes/modules from non-main Ractors (RuntimeError)
-#
```

Comment:
It doesn't seem this example demonstrated anything useful :thinking:

Reply:
It demonstrates accessing a non-sharable instance variable in a module or class raises an exception.

Reply:
Oh, but now I see how this example is useful :) The difference with the previous one is exactly in passing the class vs accessing it as a constant :)